package.json

```diff
@@ -64,24 +60,33 @@
     "@angular-eslint/template-parser": "19.2.1",
     "@angular/cli": "19.2.3",
     "@angular/compiler-cli": "19.2.1",
+    "@eslint/compat": "^1.2.7",
+    "@eslint/js": "^9.22.0",
+    "@prettier/plugin-xml": "^3.2.2",
     "@types/bootstrap": "^4",
     "@types/diff-match-patch": "1.0.36",
     "@types/jasmine": "5.1.5",
     "@types/node": "22.13.5",
-    "@typescript-eslint/eslint-plugin": "^7.18.0",
-    "@typescript-eslint/parser": "^7.18.0",
+    "@typescript-eslint/eslint-plugin": "^8.26.1",
+    "@typescript-eslint/parser": "^8.26.1",
+    "@typescript-eslint/utils": "^8.26.0",
```

Comment:
Komen deze niet al mee als peer dependencies?

Reply:
Ik moet altijd `--legacy-peer-deps` meegeven bij `npm install` of `npm ci`. Misschien is dat de reden dat het weghalen van `@typescript-eslint/parser` niet werkt. Als ik het probeer, dan falen de Karma testen. Ik heb `@typescript-eslint/utils` wel weggehaald zonder dat de testen faalden.

Reply:
Hm, linten ging toch weer fout omdat `@typescript-eslint/parser` niet gevonden kon worden. Nu staat hij weer in package.json en het werkt weer. Ik begrijp niet helemaal wat hier gebeurt, maar het ziet ernaar uit dat deze dependencies expliciet nodig zijn.